localisation/english/rikk_mdlc_l_english_world_machine_upgrades.yml

```diff
@@ -13,7 +13,7 @@
  
  # warm frozen planet
  rikk_mdlc_decision_world_machine_defrost_cryosphere: "Defrost Cryosphere"
- rikk_mdlc_decision_world_machine_defrost_cryosphere_desc: "Using a large amount of energy, we will activate a planetary geodynamo to begin warming the planet."
+ rikk_mdlc_decision_world_machine_defrost_cryosphere_desc: "Using a large amount of energy, and reversed geoelectric harvesting devices, we can begin warming the planet."
```

Comment:
Thanks for working on this. Please update to:

    A large amount of energy will be applied over time as heat to thaw enough of the planet to permit installation of a World-Machine Core.